tests/test_api_lib_policy.py

```diff
@@ -1548,6 +1549,62 @@ def test_18b_hide_audit_columns(self):
         # delete user policy
         delete_policy("hide_audit_columns_user")
 
+    def test_18c_hide_tokeninfo(self):
+        builder = EnvironBuilder(method='POST',
+                                 headers={})
+        env = builder.get_environ()
+        req = Request(env)
+        g.policy_object = PolicyClass()
+
+        # set a policy to hide the "tokenkind" and the "unknown" tokeninfo values
+        set_policy(name="hide_tokeninfo_admin",
+                   scope=SCOPE.ADMIN,
+                   action="{0!s}=tokenkind unknown".format(ACTION.HIDE_TOKENINFO))
+        g.logged_in_user = {"username": "admin1",
+                            "realm": "",
+                            "role": "admin"}
+        req.all_data = {}
+        req.User = User("Unknown")
+        hide_tokeninfo(req)
+        # Check if the hidden_tokeninfo entry was added to the request.data
+        self.assertIn('hidden_tokeninfo', req.all_data, req.all_data)
+        for col in ["tokenkind", "unknown"]:
+            self.assertIn(col, req.all_data.get("hidden_tokeninfo"))
+        # check that it won't be added when logged in as a user
+        g.logged_in_user = {"username": "user1",
+                            "realm": "",
+                            "role": "user"}
+        req.all_data = {}
+        hide_tokeninfo(req)
+        self.assertNotIn('hidden_tokeninfo', req.all_data, req.all_data)
+
+        delete_policy("hide_tokeninfo_admin")
+
+        # set a policy to hide the "number" and the "realm" columns in the audit response
```

Comment:
This is the old comment from the audit hide.

Reply:
Done in 6303d623